documentation/src/test/java/example/ClassTemplateDemo.java

```diff
@@ -52,9 +52,9 @@ void wellKnown() {
 	}
 
 	// end::user_guide[]
-	static
+	public
 	// tag::user_guide[]
-	public class MyClassTemplateInvocationContextProvider
```

Comment:
@marcphilipp might missconfig of check, but as convention is the best config, might consider rewrite migration dropping the config burden promoting into 1LOC being a dedicated recipe, fully maintained by the community.



- https://github.com/junit-team/junit-framework/commit/1536aa39502ccd229c5ef51aa90adf2c51a0491c